decidim-comments/app/forms/decidim/comments/comment_form.rb

```diff
@@ -17,6 +19,7 @@ class CommentForm < Form
       validates :alignment, inclusion: { in: [0, 1, -1] }, if: ->(form) { form.alignment.present? }
 
       validate :max_depth
+      validate :commentable_can_have_comments
```

Comment:
We need to add a spec in `comment_form_spec.rb` to check this condition

Reply:
Done